master/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -1,9 +1,14 @@
 ---
 title: Kubeadm Hosted Install
+redirect_from: latest/getting-started/kubernetes/installation/hosted/kubeadm/index
```

Comment:
Sorry, should have looked at the master file before copying over to it.... Just updated again.